conf/messages.en

```diff
@@ -118,6 +136,138 @@ entities.protector = Protectors
+individualProtector.passportDetails.expiryDate = Expiry date
+individualProtector.passportDetails.expiryDate.hint = Expiry date can be in the past. For example, 31 3 1980
+individualProtector.passportDetails.expiryDate.error.required.all = Enter the passport expiry date
+individualProtector.passportDetails.expiryDate.error.required.two = Passport expiry date must include a {0} and {1}
+individualProtector.passportDetails.expiryDate.error.required = Passport expiry date must include {0}
+individualProtector.passportDetails.expiryDate.error.invalid = Enter a real date when the individual’s passport expires
+individualProtector.passportDetails.expiryDate.error.future = Passport expiry date must be after 31 December 1499 and before 01 January 2100
+individualProtector.passportDetails.expiryDate.error.past = Passport expiry date must be after 31 December 1499 and before 01 January 2100
+
+individualProtector.idCardDetailsYesNo.checkYourAnswersLabel = Do you know {0}’s ID card details?
+individualProtector.idCardDetailsYesNo.title = Do you know the protector’s ID card details?
+individualProtector.idCardDetailsYesNo.heading = Do you know {0}’s ID card details?
+individualProtector.idCardDetailsYesNo.error.required = Select yes if you know the protector’s ID card details
+
+individualProtector.idCardDetails.checkYourAnswersLabel = What are {0}’s ID card details?
+individualProtector.idCardDetails.title = What are the protector’s ID card details?
+individualProtector.idCardDetails.heading = What are {0}’s ID card details?
+individualProtector.idCardDetails.country = ID card country of issue
+individualProtector.idCardDetails.country.error.required = Enter the protector’s ID card country of issue
+individualProtector.idCardDetails.country.error.length = The protector’s ID card country must be 100 characters or less
+individualProtector.idCardDetails.number = ID card number
+individualProtector.idCardDetails.number.hint = For example, 502135326
+individualProtector.idCardDetails.number.error.required = Enter the protector’s ID card number
+individualProtector.idCardDetails.number.error.length = The protector’s ID card number must be 30 characters or less
+individualProtector.idCardDetails.number.error.invalid = ID card number can only include letters and numbers
+individualProtector.idCardDetails.expiryDate = Expiry date
+individualProtector.idCardDetails.expiryDate.hint = Expiry date can be in the past. For example, 31 3 1980
+individualProtector.idCardDetails.expiryDate.error.required.all = Enter the ID card expiry date
+individualProtector.idCardDetails.expiryDate.error.required.two = ID card expiry date must include a {0} and {1}
+individualProtector.idCardDetails.expiryDate.error.required = ID card expiry date must include a {0}
+individualProtector.idCardDetails.expiryDate.error.invalid = Enter a real date when the individual’s ID card expires
```

Comment:
Again, using 'individual' when 'protector' has been used elsewhere. May be worth double checking with Rachel

Reply:
These are the same format settlor uses so I'm assuming they are correct?

Reply:
Could you double check with Rachel please? Doesn't seem right to refer to 'the protector' in all other error messages and then 'the individual' in this one